src/components/app-configure/AppChain.vue

```diff
@@ -126,11 +126,12 @@ export default {
     }
 
     function changeChainType(ev, selectedChainType) {
-      if (ev.value) {
-        store.dispatch(env.value + "/updateChainType", selectedChainType);
-        if (props.isConfigured && !store.getters.onConfigChange) {
-          store.dispatch("configChangeDetected");
-        }
+      store.dispatch(
+        env.value + "/updateChainType",
+        ev.value ? selectedChainType : ""
+      );
+      if (props.isConfigured && !store.getters.onConfigChange) {
+        store.dispatch("configChangeDetected");
       }
     }
```

Comment:
We could incorporate a few small changes to make the code more readable.

- Rename `ev` to `event`. It's not immediately obvious what `ev` refers to and it's too similar to the existing `env` variable.
- Rename `selectedChainType` to `chainType` as this event handler will also run when a user _unselects_ a chain type.
- Create an `isSelected` variable to clearly point out what `event.value` is referring to.

```js
function changeChainType(event, chainType) {
  const isSelected = event.value
  store.dispatch(
    env.value + "/updateChainType",
    isSelected ? chainType : ""
  )
  ...
}
```

Reply:
@ajithranka refactored code.